VMs/metrics/ubuntu16.04/env.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright (c) 2018 Intel Corporation
+# 
+# SPDX-License-Identifier: Apache-2.0
+
+PRIVKEYNAME=$(pwd)/virsh.rsa
+PUBKEYNAME=${PRIVKEYNAME}.pub
+
+VM_NAME=ubnt16.04_master
+VM_CLONE_NAME=ubnt16.04_clone
+IMG_BASE=${IMG_DIR}/xenial-server-cloudimg-amd64-disk1.img
```

Comment:
Hard-coded architecture. Could you change this to use `$(dpkg --print-architecture)` as this file is Ubuntu-specific?